zenml/core/pipelines/base_pipeline.py

```diff
@@ -387,23 +343,19 @@ def run_config(self, config: Dict[Text, Any]):
         Args:
             config: dict of ZenML config.
         """
-        if OrchestratorLocalBackend.BACKEND_KEY not in self.backends_dict:
-            raise AssertionError('Orchestrator backend missing!')
-        orch_backend = self.backends_dict[OrchestratorLocalBackend.BACKEND_KEY]
-
-        # TODO: [LOW] Make sure this is subclass of OrchestratorLocalBackend
-        orch_backend.run(config)
+        assert issubclass(self.backend.__class__, OrchestratorLocalBackend)
+        self.backend.run(config)
 
     @track(event=RUN_PIPELINE)
     def run(self,
-            backends: Optional[List[BaseBackend]] = None,
+            backend: OrchestratorLocalBackend = None,
```

Comment:
I think this "local" backend here also serves as the base backend for the non-local variants, however having the word local in the naming is a bit confusing. At first, I thought that the "run" method of the "basepipeline" would always accept a local-type orchestrator.

Reply:
changed the names accordingly to base